### ISSUE TITLE
Exclude `http://localhost[:port]` from secure URL in readme logic

### ISSUE THREAD

**Issue by kevmoo** [dart-lang/pub-dev#3880]
URL: https://pub.dev/packages/redux/score

![Screen Shot 2020-07-23 at 4 21 12 PM](https://user-images.githubusercontent.com/17034/88347839-9b743600-cd00-11ea-8e6e-18051c1af621.png)

I guess this might be a pana issue...

**Comment by isoos** [comment]
Yes, that's pana. We shall address it after the ongoing in that codebase.
